content-src/experiments/lockbox.yaml

```diff
@@ -0,0 +1,53 @@
+dev: true
+id: 999
+title: 'Lockbox'
+slug: lockbox
+platforms: ['addon']
+locales:
+- 'en-US'
+locale_grantlist:
+- 'en-US'
+min_release: 57
+thumbnail: /static/images/experiments/notes/experiments_experiment/thumbnail.png
```

Comment:
Do we want to put in Lockbox specific assets now, or are we waiting for a future PR?